src/sensor/sensor.py

```diff
@@ -0,0 +1,102 @@
+import uuid
+import psutil
+import json
+import time
+import socket
+import sys
+import pickle
+from functools import reduce
+sys.path.append("../../")
+print(sys.path)
+from src.common.TCP_messages import *
+
+
+BUFFER_SIZE = 1024
+QUIET_MODE = False
+
+def prepare_metadata(metadata):
+    fields={
+            "CPU Core count": lambda: psutil.NUM_CPUS,
+            "Total RAM": lambda: psutil.virtual_memory().total
+            }
+    result = []
+    for key in metadata:
+        if key in fields.keys():
+            result.append({"key": key, "value": fields[key]()})
+    return result
+
+def prepare_register(data, sensor_id = ""):
+    identifier = str(uuid.uuid4()) if sensor_id == "" else sensor_id
```

Comment:
"identifier" ma być unikalny w każdej rejestracji dla Sensora. Jeśli nie jesteśmy w stanie wyciągnąć adresu MAC maszyny, to dorzućmy kolejny klucz do pliku json, w którym trzeba będzie ustawić "identifier".
Na jednym komputerze może być uruchomiony tylko jeden Sensor. Takie jest założenie i dlatego adres MAC wydawał się dobrym unikalnym identyfikatorem komputera